Components/badge/badge.css

```diff
@@ -0,0 +1,58 @@
+.badge{
+    position: relative;
+    display: inline-block;
+}
+
+.indicator{
+    border-radius: 50%;
+    background-color: var(--INDICATOR-GREEN);
+    position: absolute;
+    
+    border: 2px solid var(--BACKGROUND-COLOR);
+}
+
+.indicator-sm{
+    height: 1rem;
+    width: 1rem;
+    bottom: 0.1rem;
+    left: 1.3rem;
+}
+
+.indicator-m{
+    height: 1.5rem;
+    width: 1.5rem;
+    bottom: 0.2rem;
+    left: 2.5rem;
+}
+
+.indicator-l{
+    height: 2rem;
+    width: 2rem;
+    bottom: 0.3rem;
+    left: 3.6rem;
+}
+
+.indicator-xl{
+    height: 2.5rem;
+    width: 2.5rem;
+    bottom: 0.4rem;
+    left: 4.8rem;
+}
+
+.badge-icon{
+    width: 4rem;
+    height: 4rem;
+}
+
+.text-badge{
+    width: 2.5rem;
+    height: 2.5rem;
+    padding: 1rem;
+    background-color: red;
```

Comment:
As, you are using variables for naming colors then keep it consistent. And to name variable, it will nice to name it like `--COLOR-BACKGROUND`.

Reply:
Yeah, forgot to change this. Will do. Will change the name too.